src/main/java/frc/robot/commands/intake/RunIntake.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig;
+import frc.robot.subsystems.intake.Intake;
+
+public class RunIntake extends Command {
+  private final Intake m_intake;
+
+  /**
+   * Creates a new RunIntake command, which runs the roller motor on the intake subsystem to intake
+   * a note
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public RunIntake(Intake subsystem) {
```

Comment:
nit: we should bias towards using descriptive names for variables. Something like
```
public RunIntake(Intake intake) { ...
```

would be more clear.

Reply:
Changed name in commit